pset-functions.js

```diff
@@ -136,6 +191,48 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+const word = "javascript"
+
+const scoreKey = {
+  a: 1,
+  e: 1,
+  i: 1,
+  o: 1,
+  u: 1,
+  l: 1,
+  n: 1,
+  r: 1,
+  s: 1,
+  t: 1,
+  d: 2,
+  g: 2,
+  b: 3,
+  c: 3,
+  m: 3,
+  p: 3,
+  f: 4,
+  h: 4,
+  v: 4,
+  w: 4,
+  y: 4,
+  k: 5,
+  j: 8,
+  x: 8,
+  q: 10,
+  z: 10,
+}
+
+function scrabbleScore (word) {
+  const wordSplit = word.split('')
+
+  const scoreArray = wordSplit.map(letter => scoreKey[letter])
+  const totalScore = scoreArray.reduce((a,b) => a + b)
```

Comment:
@kcycle impressive use of `.map()` and `.reduce()` to calculate the total score associated with each word